CHANGELOG.rst

```diff
@@ -1,5 +1,22 @@
 .. _changelog:
 
+0.54.2
+------
+
+Renku ``0.54.2`` fixes a bug with testing the cloud storage connection for WebDAV.
+
+User-Facing Changes
+~~~~~~~~~~~~~~~~~~~
+
+**🐞 Bug Fixes**
+
+- **Data Services**: Correctly obscure rclone values when testing cloud storage connections
```

Comment:
@Panaetius Is this a user facing change? My expectation is that changelog items that are in the User Facing changes section should also be user-comprehensible, and I'm not sure what this means. Maybe we need to add a little more info, to explain what the user impact is? (I assume it's related to the WebDAV thing above?)

Reply:
I changed the wording.

Reply:
Perfect, thank you!